src/DataCollection.UWP/Views/AddFeatureView.xaml

```diff
@@ -32,43 +32,21 @@
         <Rectangle
             Grid.Row="0"
             Grid.Column="0"
-            Fill="Black"
-            Opacity=".5" />
+            Grid.ColumnSpan="3"
+            Fill="{ThemeResource SystemControlChromeHighAcrylicElementMediumBrush}" />
         <Rectangle
             Grid.Row="1"
             Grid.Column="0"
-            Fill="Black"
-            Opacity=".5" />
-        <Rectangle
-            Grid.Row="2"
-            Grid.Column="0"
-            Fill="Black"
-            Opacity=".5" />
-        <Rectangle
-            Grid.Row="0"
-            Grid.Column="1"
-            Fill="Black"
-            Opacity=".5" />
-        <Rectangle
-            Grid.Row="2"
-            Grid.Column="1"
-            Fill="Black"
-            Opacity=".5" />
-        <Rectangle
-            Grid.Row="0"
-            Grid.Column="2"
-            Fill="Black"
-            Opacity=".5" />
+            Fill="{ThemeResource SystemControlChromeHighAcrylicElementMediumBrush}" />
```

Comment:
I think I had suggested acrylic for this in the past, but I'm remembering now that there is an issue with seams when two acrylic surfaces are next to each other: 

![ApplicationFrameHost_2019-05-13_09-15-02](https://user-images.githubusercontent.com/29742178/57638104-ea9dd300-7561-11e9-88fd-d42284af5041.jpg)

If there is a way to get a semitransparent non-acrylic surface that would probably be better, ideally matching the user's light/dark setting.

Reply:
I used a border instead. I don't know why I didn't think of that option before.